### PR TITLE
created routes

### PR DESCRIPTION
**Pull request by DavidKhanTech**
User routes created

### CODE REVIEW

File: routes/users-routes.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { Router } = require('express')
+const controllers = require("../controllers/users-controller")
+
+const router = Router()
+
+router.get('models/user', controllers.getUsers)
```
Comment (indianakuffer) [inline routes/users-routes.js]: should be '/users' with an s

File: routes/users-routes.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+router.get('/model/user:id', controllers.getUser)
```
Comment (indianakuffer) [inline routes/users-routes.js]: where there's an :id slug it needs to have a slash in front of it
'/user/:id'

**Review by indianakuffer — CHANGES_REQUESTED** [review]
see comments, and all endpoints should be plural

File: routes/users-routes.js (hunk 1 above)
Comment (DavidKhanTech) [reply routes/users-routes.js]: done

**Comment by DavidKhanTech** [comment]
fixed the bugs, pushed another commit,  thank you!

**Review by indianakuffer — APPROVED** [review]
Approved, comments look accounted for!

**Pull request merged**